src/plugins/lanplus/lanplus.c

```diff
@@ -1364,56 +1341,61 @@ void getIpmiPayloadWireRep(
 	} else {
 		bridgedRequest = 1;
 
-		if(intf->transit_addr != ourAddress && intf->transit_addr != 0)
-		{
+		if (intf->transit_addr != ourAddress
+		    && intf->transit_addr != 0) {
 			bridgedRequest++;
 		}
 		/* bridged request: encapsulate w/in Send Message */
 		cs = len;
 		msg[len++] = IPMI_BMC_SLAVE_ADDR;
 		msg[len++] = IPMI_NETFN_APP << 2;
 		tmp = len - cs;
-		msg[len++] = ipmi_csum(msg+cs, tmp);
+		msg[len++] = ipmi_csum(msg + cs, tmp);
 		cs2 = len;
 		msg[len++] = IPMI_REMOTE_SWID;
 		msg[len++] = curr_seq << 2;
 
-		
-		msg[len++] = 0x34;			/* Send Message rqst */
-		if(bridgedRequest == 2)
-			msg[len++] = (0x40|intf->transit_channel); /* Track request*/
+
+		msg[len++] = 0x34; /* Send Message rqst */
+		if (bridgedRequest == 2)
+			msg[len++] =
+				(0x40
+				 | intf->transit_channel); /* Track request*/
 		else
-			msg[len++] = (0x40|intf->target_channel);    /* Track request*/
-			
+			msg[len++] =
```

Comment:
This can probably be cleaned by moving the comment from the end of the line to immediately above and adding curly braces.

Reply:
I guess that must be done then.